cookieninja/extensions.py

```diff
@@ -1,9 +1,11 @@
 """Jinja2 extensions."""
+import arrow
```

Comment:
This is not a built in package, reorder imports

Reply:
Done!